### DISCUSSION TITLE
后端服务器全部失效，无法削刮了。

### DISCUSSION THREAD

**Discussion by wanbo1** [metatube-community/jellyfin-plugin-metatube#254] [General]
后端服务器全部失效，无法削刮了。

**Comment by thomascst** [comment]
我也是，現在搜之前有削刮的id 都沒有result

**Comment by AnsonAiTRAY** [comment]
同样，自己搭的服务器也不行了

**Comment by wanbo1** [comment]
用Koyeb，搭建后端，可以的，试过了。
教程在此：

> https://lrqcu.com/forum.php?mod=viewthread&tid=1773888&highlight=

不知道你们能不能打开。

**Reply by AnsonAiTRAY** [reply]
我就是koyeb搭的后端，刮削不全的，fanza上面的刮不到，里番以前是能刮削的现在刮不了了

**Reply by wanbo1** [reply]
那只能另外用Mdcx削刮了。

**Reply by AnsonAiTRAY** [reply]
其他的主要是没用过，弄起来怕弄不明白很浪费时间，我感觉可能是fanza更新了什么反爬机制，不知道作者能不能更新一下

**Reply by wanbo1** [reply]
我试了下好像可以削刮动画的
![Uploading 搜狗截图24年01月04日0015_1.jpg…]()

**Reply by wanbo1** [reply]
![image](https://github.com/metatube-community/jellyfin-plugin-metatube/assets/155458710/40d1499c-6a63-4d6d-aff2-f52c3413307c)

**Reply by AnsonAiTRAY** [reply]
想问一下你是用番号（图上那个amc什么的）搜还是直接用名字搜的，而且图上好像还是没有fanza的刮削，以前一般都会有fanza的出来

**Reply by gzk110** [reply]
> 用Koyeb，搭建后端，可以的，试过了。 教程在此：
> https://lrqcu.com/forum.php?mod=viewthread&tid=1773888&highlight=
> 不知道你们能不能打开。

打开是个购买域名的网站

**Comment by xjasonlyu** [comment]
没有失效，DMM/Fanza 锁区日本ip，所以请使用日本节点的VPS，不然无法搜索出任何数据。

**Reply by AnsonAiTRAY** [reply]
是搭建后端的ip地址要在日本是吗

**Reply by xjasonlyu** [reply]
对的

**Comment by Yustyn555** [comment]
> 用Koyeb，搭建后端，可以的，试过了。
> 教程在此：
> 
> > https://lrqcu.com/forum.php?mod=viewthread&tid=1773888&highlight=
> 
> 不知道你们能不能打开。
> 
> 
> 

我按照教程搭建了，但填入URL，按保存后就像重启服务器一样，URL和token都没有填上，不知道撒问题。

**Reply by foxyuaaalll** [reply]
我也遇到这个问题，你解决了吗

**Reply by Yustyn555** [reply]
现在可以了，更新版本

**Reply by foxyuaaalll** [reply]
请问是更新EMBY还是metatube呢

**Reply by Yustyn555** [reply]
都更新，新插件支持4.8的版本

**Reply by foxyuaaalll** [reply]
搞定了，谢谢

**Reply by Mrjind** [reply]
你好，更新了最新版本后，文字信息显示了，但是图片刮不到有这种情况出现吗

**Comment by wanbo1** [comment]
手动识别，如果能识别出来，在插件设置里把能削挂出图片的网站放到第一个的位置。



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mrjind ***@***.***&gt;
发送时间: 2024年6月8日 00:31
收件人: metatube-community/jellyfin-plugin-metatube ***@***.***&gt;
抄送: wanbo1 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [metatube-community/jellyfin-plugin-metatube] 后端服务器全部失效，无法削刮了。 (Discussion #254)





 
你好，更新了最新版本后，文字信息显示了，但是图片刮不到有这种情况出现吗

**Reply by Mrjind** [reply]
我用的Metatube，似乎不能且没有更改网站的设置，识别的话在日志里把地址复制出来，翻墙可以看到图片，这种情况似乎是要走代理了，是需要设置旁路由这样吗

**Comment by wanbo1** [comment]
在插件设置里，有个地方是削刮网址排序的，调整前后位置，把javbus放第一个





发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mrjind ***@***.***&gt;
发送时间: 2024年6月8日 00:52
收件人: metatube-community/jellyfin-plugin-metatube ***@***.***&gt;
抄送: wanbo1 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [metatube-community/jellyfin-plugin-metatube] 后端服务器全部失效，无法削刮了。 (Discussion #254)





 
我用的Metatube，似乎不能且没有更改网站的设置，识别的话在日志里把地址复制出来，翻墙可以看到图片，这种情况似乎是要走代理了，是需要设置旁路由这样吗

**Reply by Mrjind** [reply]
没有啊，你用的Koyeb吗，这个后端现在不翻墙是访问不了了的

**Comment by wanbo1** [comment]
削刮一定要翻墙的。我用的jellyfin，Koyeb这个后端不是很稳定，大概几个小时候就会停止运行，要手动重新部署。





插件，削刮网站顺序排列有先后。
------------------ 原始邮件 ------------------
发件人:                                                                                                                        "metatube-community/jellyfin-plugin-metatube"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月8日(星期六) 中午12:50
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [metatube-community/jellyfin-plugin-metatube] 后端服务器全部失效，无法削刮了。 (Discussion #254)





 
没有啊，你用的Koyeb吗，这个后端现在不翻墙是访问不了了的
